.github/workflows/ci.yml

```diff
@@ -15,7 +15,7 @@ jobs:
           sudo apt-get install jq
           echo "::set-output name=matrix::$(bash scripts/get-all-test-paths.sh)"
         env:
-          CHANGED_FILES: ${{ steps.changed-files.outputs.added_modified }}
+          CHANGED_FILES: ${{ steps.changed-files.outputs.all }}
```

Comment:
Why did you change this?

Reply:
Because I noticed that half of the tests did not run (for example if I delete/rename stuff that is breaking something nobody would notice.)